practice-two/src/components/FormInput/FormInput.css

```diff
@@ -0,0 +1,34 @@
+.form__input {
+  display: flex;
+  flex-direction: column;
+  width: 520px;
+  margin-bottom: 14px;
+}
+
+.input__field {
+  padding: 15px;
+  border-radius: 5px;
+  border: 1px solid var(--cl-gray);
+}
+
+.input__label {
+  font-size: var(--fs-sm);
+  color: var(--cl-secondary);
+  font-weight: var(--fw-lg);
+  margin-bottom: 10px;
+}
+
+.error__message {
+  font-size: var(--fs-xx-sm);
+  margin: 3px;
+  color: var(--cl-error);
+  display: none;
+}
+
+.input__field:invalid[focused='true'] {
+  border: 1px solid var(--cl-error);
+}
+
+.input__field:invalid[focused='true'] ~ .error__message {
+  display: inline;
```

Comment:
span is inline tag why need to set **display: inline;**

Reply:
I used to show the errorMessage is hidden by display: none
But I did not use display anymore
I decided to use visibility to hidden and show errorMessage
![image](https://github.com/datdragon777/React-Training/assets/74806826/aee4684d-9c2e-439d-b664-01a64990f306)

Reply:
you have a condtion if have the error message will render the message or not. So no need any style for display

Reply:
Yes, I update as you commented
![image](https://github.com/datdragon777/React-Training/assets/74806826/eac8cb73-f0fc-42b7-94ab-1517836d967c)

![image](https://github.com/datdragon777/React-Training/assets/74806826/7770d756-2dda-4082-81b7-b9614947167e)